tools/ci_build/github/linux/docker/scripts/training/ortmodule/stage1/requirements.txt

```diff
@@ -1,15 +1,14 @@
 --pre
--f https://download.pytorch.org/whl/nightly/cu101/torch_nightly.html
 # transformers requires sklearn
 pandas
 sklearn
 numpy==1.19.5
 transformers==v4.3.2
-torch
-torchvision
-torchtext
+torch==1.8.1+cu102
+torchvision==0.9.1+cu102
+torchtext==0.9.1
 tensorboard==v2.0.0
 h5py
 wget
 # PyTorch Lightning (nightly) is used for CI tests only
-https://github.com/PyTorchLightning/pytorch-lightning/archive/master.zip
\ No newline at end of file
+https://github.com/PyTorchLightning/pytorch-lightning/archive/master.zip
```

Comment:
Did you check whether lightning has a stable version that depends on torch 1.8.1?

Reply:
Using version 1.2.5.